llvm/include/llvm/IR/RepoTicket.h

```diff
@@ -59,46 +58,55 @@ struct GONumber {
 
 /// A structure of a global object (GO) information.
 struct GOInfo {
-  /// GO's initial hash value which does not include the hash of its dependents.
+  /// The Initial-digest is the hash of an object (a function or global
+  /// variable) without consideration for any references to other objects.
   DigestType InitialDigest;
-  /// A set of global objects which the GO's hash depenendent on.
-  DependenciesType Dependencies;
-  /// A set of global variables which the GO's hash contributed to.
-  ContributionsType Contributions;
+  /// The Contributions of an object X are those objects Y which transitively
```

Comment:
Is object X the one that this GOInfo applies to? I think this would be clearer that this you are defining them by putting them in brackets:
"/// The Contributions of an object (X) are those objects (Y) which transitively..."

Reply:
> 
> Is object X the one that this GOInfo applies to? 
>
Yes.
>
> "/// The Contributions of an object (X) are those objects (Y) which transitively..."
>
Fixed, thanks.